src/matlab2tikz.m

```diff
@@ -4500,6 +4500,34 @@ function warnFacetedInterp(m2t)
     end
 end
 % ==============================================================================
+function   cbarStyleOptions = getColotBarTicks(handle, cbarStyleOptions)
+if strcmp( get(handle,'TickLabelsMode') , 'manual')
+
```

Comment:
Please generate an opts structure as we do in the rest of the code. That makes things much easier